pkg/controllers/deprovisioning/consolidation.go

```diff
@@ -93,17 +91,17 @@ func (c *consolidation) markConsolidated() {
 }
 
 // ShouldDeprovision is a predicate used to filter deprovisionable nodes
-func (c *consolidation) ShouldDeprovision(_ context.Context, cn *Candidate) bool {
-	if val, ok := cn.Annotations()[v1alpha5.DoNotConsolidateNodeAnnotationKey]; ok {
-		c.recorder.Publish(deprovisioningevents.Unconsolidatable(cn.Node, cn.Machine, fmt.Sprintf("%s annotation exists", v1alpha5.DoNotConsolidateNodeAnnotationKey))...)
-		return val != "true"
+func (c *consolidation) shouldDeprovision(_ context.Context, cn *Candidate, isEmptyConsolidation bool) bool {
+	if cn.Annotations()[v1alpha5.DoNotConsolidateNodeAnnotationKey] == "true" {
```

Comment:
We've changed this behavior, and will need a new test. Consider punting on this change for a future PR.

Reply:
I'm opting to call it out in the PR description. Frankly, this is the correct behavior to begin with so this change seems reasonable to me

Reply:
I'll call it out in the PR description since this is technically the way that this behavior should have been previously.